documentation/reference/document-select-language.html

```diff
@@ -178,7 +186,45 @@ <h3 id="comparison">Comparison</h3>
 </table>
 </p>
 
-<h4>Locale / Character sets</h4>
+<h4 id="null-value-comparisons">Comparisons with missing fields (null values)</h4>
```

Comment:
id is a function of heading - in this case should be _comparisons-with-missing-fields-null-values_

Reply:
Fixed